src/dev/build/tasks/os_packages/docker_generator/templates/ironbank/hardening_manifest.yaml

```diff
@@ -1,21 +1,17 @@
 ---
 apiVersion: v1
-
 # The repository name in registry1, excluding /ironbank/
 name: 'elastic/kibana/kibana'
-
 # List of tags to push for the repository in registry1
 # The most specific version should be the first tag and will be shown
 # on ironbank.dsop.io
 tags:
   - '{{version}}'
   - 'latest'
-
 # Build args passed to Dockerfile ARGs
 args:
   BASE_IMAGE: 'redhat/ubi/ubi9'
-  BASE_TAG: '9.4'
-
+  BASE_TAG: "9.4"
```

Comment:
It seems others use `double quotes`:
- https://github.com/elastic/apm-server/blob/main/packaging/ironbank/hardening_manifest.yaml#L17
- https://github.com/elastic/elasticsearch/blob/main/distribution/docker/src/docker/iron_bank/hardening_manifest.yaml#L17
- https://github.com/elastic/beats/blob/main/dev-tools/packaging/templates/ironbank/filebeat/hardening_manifest.yaml#L17
-